lib/central_mail/utilities.rb

```diff
@@ -22,6 +16,21 @@ module Utilities
     MISSING_ZIP_CODE_ERROR_MSG = 'Missing ZIP Code. ZIP Code must be 5 digits, ' \
                                  'or 9 digits in XXXXX-XXXX format. Specify \'00000\' for non-US addresses.'
 
+    # valid lines of business.  From 'LOG' on these keys must be mapped to 'CMP' until they are formally supported.
+    # See https://vajira.max.gov/browse/API-678 for a description of the LOBs
+    DEFAULT_VALID_LOB = { 'CMP' => 'CMP', 'PMC' => 'PMC', 'INS' => 'INS', 'EDU' => 'EDU', 'VRE' => 'VRE',
+                          'BVA' => 'BVA', 'FID' => 'FID', 'NCA' => 'NCA', 'OTH' => 'CMP' }.freeze
+    DEFAULT_VALID_LOB_MSG = { 'CMP' => 'CMP', 'PMC' => 'PMC', 'INS' => 'INS', 'EDU' => 'EDU', 'VRE' => 'VRE',
+                              'BVA' => 'BVA', 'FID' => 'FID', 'NCA' => 'NCA', 'OTH' => 'OTH' }.freeze
```

Comment:
On second glance, there are a couple of things here that we could probably clean up. The first is this comment:
`# valid lines of business.  From 'LOG' on these keys must be mapped to 'CMP' until they are formally supported.`

I think it can be removed entirely because 'LOG' isn't even a key in the list, at least not anymore.

It looks like we also might be able to remove `DEFAULT_VALID_LOB_MSG`? The only place that I see it being used is the `UploadValidator`, and that only looks at the keys of `DEFAULT_VALID_LOB_MSG`, which happen to match the keys of `DEFAULT_VALID_LOB` exactly.

Reply:
Ok, I've updated this and it looks like PR checks are working again - thanks!